pricelist_brand/models/product_pricelist.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    def _compute_price_rule_get_items(
+        self, products_qty_partner, date, uom_id, prod_tmpl_ids, prod_ids, categ_ids
+    ):
+        self.ensure_one()
+        product_tmpls = self.env["product.template"].browse(prod_tmpl_ids)
+        brand_ids = product_tmpls.mapped("product_brand_id").ids
+        # Load all rules
+        self.env["product.pricelist.item"].flush(["price", "currency_id", "company_id"])
+        self.env.cr.execute(
+            """
+            SELECT
+                item.id
+            FROM
+                product_pricelist_item AS item
+            LEFT JOIN product_category AS categ ON item.categ_id = categ.id
+            LEFT JOIN product_brand AS brand ON item.product_brand_id = brand.id
+            WHERE
+                (item.product_tmpl_id IS NULL OR item.product_tmpl_id = any(%s))
+                AND (item.product_id IS NULL OR item.product_id = any(%s))
+                AND (item.categ_id IS NULL OR item.categ_id = any(%s))
+                AND (item.product_brand_id IS NULL OR item.product_brand_id = any(%s))
+                AND (item.pricelist_id = %s)
+                AND (item.date_start IS NULL OR item.date_start<=%s)
+                AND (item.date_end IS NULL OR item.date_end>=%s)
```

Comment:
Hi, I think a line 

`AND (item.active = TRUE)` 

is missing if compared to the original method. Has it been removed for specific reasons?

https://github.com/odoo/odoo/blob/14.0/addons/product/models/product_pricelist.py#L111